torch/csrc/jit/passes/quantization.cpp

```diff
@@ -766,6 +801,10 @@ graph(%self, %a, %b, %inplace):
 
   const std::vector<std::reference_wrapper<const PatternInfo>> delay_patterns =
       {
+          nn_linear_nn_relu,
+          nn_linear_f_relu,
+          f_linear_nn_relu,
+          f_linear_f_relu,
```

Comment:
this looks like a great naming convention, we should probably follow the same for other patterns in the list.

Reply:
Will do when adding it to conv